hnn_core/network.py

```diff
@@ -57,6 +57,87 @@ def read_spikes(fname, gid_dict=None):
     return Spikes(times=spike_times, gids=spike_gids, types=spike_types)
 
 
+def _create_coords(n_pyr_x, n_pyr_y, n_common_feeds, p_unique_keys,
+                   zdiff=1307.4):
+    """Creates coordinate grid.
+
+    Parameters
+    ----------
+    n_pyr_x : int
+        The number of Pyramidal cells in x direction.
+    n_pyr_y : int
+        The number of Pyramidal cells in y direction.
+    n_common_feeds : int
+        The number of common feeds.
+    p_unique_keys : list of str
+        The keys of the dictionary p_unique.
+    zdiff : float
+        Expressed as a positive DEPTH of L5 relative to L2
+        This is a deviation from the original, where L5 was defined at 0
+        This should not change interlaminar weight/delay calculations.
+
+    Returns
+    -------
+    pos_dict : dict of list of tuple (x, y, z)
+        Dictionary containing coordinate positions.
+        Keys are 'L2_pyramidal', 'L5_pyramidal', 'L2_basket', 'L5_basket',
+        'common', and p_unique_keys
```

Comment:
Is `pos_dict['p_unique_keys']` a dictionary or list? This might be a good time to format and label the common and unique feed coordinates in a more consistent manner. Maybe use 'common_feeds' (which should actually be the key to a list that associates each tuple with a given common feed key like 'rhythmic') as the pos_dict key instead of 'common' and 'unique_feeds' instead of 'p_unique_keys'.

Reply:
sorry this is my bad. I pushed a clarification commit: https://github.com/jonescompneurolab/hnn-core/pull/122/commits/fa3c40f5c699586182100616bb426bd0847eba7d. Does this clear the air a bit? So it's not a nested dictionary. It's a flat dictionary and each dictionary element is a list.

Reply:
Yes, this helps!